base/loading.jl

```diff
@@ -425,32 +427,41 @@ julia> pkgdir(Foo)
 
 julia> pkgdir(Foo, "src", "file.jl")
 "/path/to/Foo.jl/src/file.jl"
+
+julia> pkgdir() # returns nothing because Main isn't a package module
+
 ```
 
 !!! compat "Julia 1.7"
     The optional argument `paths` requires at least Julia 1.7.
+
+!!! compat "Julia 1.9"
+    The current module (no Module argument) methods require at least Julia 1.9.
 """
 function pkgdir(m::Module, paths::String...)
     rootmodule = moduleroot(m)
     path = pathof(rootmodule)
     path === nothing && return nothing
     return joinpath(dirname(dirname(path)), paths...)
 end
+pkgdir() = pkgdir(@__MODULE__)
```

Comment:
This will just immediately be macro expanded to `Base`, or? Don't you need a macro for this type of functionality?

Reply:
oh, true. This could either become macro versions, or just a note in the docstrings that `pkgdir(@__MODULE__)` etc. works

Reply:
updated